pkg/api/schemas/language.go

```diff
@@ -0,0 +1,41 @@
+package schemas
+
+// UnifiedChatRequest defines Glide's Chat Request Schema unified across all language models
+type UnifiedChatRequest struct {
+	Message        ChatMessage   `json:"message"`
+	MessageHistory []ChatMessage `json:"messageHistory"`
+}
+
+// UnifiedChatResponse defines Glide's Chat Response Schema unified across all language models
+type UnifiedChatResponse struct {
```

Comment:
Should this be updated to match unified response in the GEP?

Reply:
The previous UnifiedChatRequest looked like this:

```go
type UnifiedAPIData struct {
	// Model          string                 `json:"model"`
	// APIKey         string                 `json:"api_key"`
	// Params         map[string]interface{} `json:"params"`
	Message        map[string]string      `json:"message"`
	MessageHistory []map[string]string    `json:"messageHistory"`
}
```

Since that schema is going to be used in Glide API to validate incoming requests, I though it would be better to have it more specific then just `map[string]string`. So I looked through other context and come up with this definition. 

I was not 100% sure it's good for us, so I wanted to double check with you if that make sense.